src/Guest.js

```diff
@@ -1,16 +1,49 @@
-class Guest {
-  constructor() {
-    this.info = info;
-    this.data = data;
-  }
+//ITERATION 2 GUEST NEEDS:
+import Overlook from './Overlook';
+//SHOULD SEE A DASHBOARD THAT SHOWS:
+
+//ALL ROOM BOOKINGS I HAVE MADE
+
+//TOTAL AMOUNT SPENT ON ROOMS
+
+//need to make an instance of guest at login matching users data from api
+// need to also load bookings at loadin to be able to compare data for guest
+// or a way to reach out and get bookings fetch
+// keep bookings fetch in scripts or main file and on click fetch data
+// how to pause to allow load like a travel site, may not need since data so much smaller
 
-  //extends from parent ?
-  // what specific things does this class do that others dont
+//methods are for through iteration 2
 
+class Guest extends Overlook {
+  constructor(rooms, bookings, id, name) {
+    super(rooms, bookings);
+    this.id = id;
+    this.name = name;
+    this.allVisits = [];
+    this.stay; 
+  }
+
+  allBookings(bookings) {
```

Comment:
I would probably name this getAllBookings or something, the verb indicates that it's a method which will be helpful when going over the code

Reply:
Cool, thanks for reviewing everything. You are right that would be more readable. Ill have to make that change right now before, I forget with the final going on .